quizcon/templates/main/student_help.html

```diff
@@ -0,0 +1,93 @@
+        <div class="modal-content">
+            <div class="modal-body">
+                <div class="row p-0 m-0">
+                    <div class="col-md-6">
+                        <div id="studentHelpCarouselText" class="studentHelpCarousel carousel slide" data-bs-ride="carousel" data-bs-interval="false" data-bs-wrap="false">
+                            <div class="carousel-inner">
+                                <div class="carousel-item carousel-txt-item text-center align-self-center active">
+                                    <h2>Before You Begin</h2>
+                                    <p>Familiarize yourself with these unique Quizzing With Confidence conventions before taking this quiz.</p>
+                                    <p>Unlike traditional multiple choice questions, the Quizzing With Confidence triangle provides a range of answer possibilities.</p>
+                                </div>
+                                <div class="carousel-item carousel-txt-item text-center align-self-center">
+                                    <h2>Scoring</h2>
+                                    <p>It's not just "correct" and "incorrect" with Quizzing With Confidence. Your score is unique calculation based on both difficulty and confidence that your instructor determines in advance.</p>
+                                    <p>In Quizzing With Confidence, you can even say "I don't know" with Option D.</p>
+                                </div>
+                                <div class="carousel-item carousel-txt-item">
+                                    <h5 class ="text-center">This Quiz</h5>
+                                    <ul>
+                                        <li>The difficulty score scheme for this quiz has been set to {{quiz.scoring_scheme_verbose}}.</li>
+                                        <li>You will be able to view the answers {{quiz.show_answers_verbose}}{% if quiz.show_answers == 2 %} {{quiz.show_answers_date}}. {% else %}.{% endif %}</li>
+                                    </ul>
+                                    <h5 class ="text-center">Try it out!</h5>
+                                    <p>The practice question on the right has <strong>A</strong> as the correct answer. As you click around, you will be able to see your potential number of points.</p>
+                                    <ul>
+                                        <li>A. Correct Answer</li>
+                                        <li>B. Incorrect Answer</li>
+                                        <li>C. Incorrect Answer</li>
+                                        <li>D. I Don't Know</li>
+                                </div>
+                                <div class="carousel-item carousel-txt-item text-center align-self-center">
+                                    <h2>Ready?</h2>
+                                    <p>When you are ready to take your quiz, <br />click <strong>Start Quiz</strong></p>
```

Comment:
I'm distracted that the "Start Quiz" button is way over on the right. Can it be within the main text area? Perhaps the right-hand side could simply be a checkmark or other icon indicating the user is finished with the wizards?